ibm/service/cis/resource_ibm_cis_domain.go

```diff
@@ -33,6 +36,12 @@ func ResourceIBMCISDomain() *schema.Resource {
 				Description: "CISzone - Domain",
 				Required:    true,
 			},
+			cisDomainType: {
+				Type:        schema.TypeString,
+				Description: "CISzone - Domain Type",
+				Default:     "full",
+				Optional:    true,
```

Comment:
Add validator funciton to allow only "full", "partial"

Reply:
Added